src/main/resources/application.yml

```diff
@@ -1,39 +0,0 @@
-spring:
```

Comment:
This should not be included in the PR.

Reply:
We removed it from .gitignore and removed it locally

Reply:
The deletion of this file still shows up in the PR.